### PR TITLE
add disable-exec.inc to all profiles with apparmor

### PR DESCRIPTION
**Pull request by smitsohu**
As discussed in #2385 and #2505. These restrictions already exist in the firejail-default `apparmor` policy, so there should be no regressions (only uncovered bugs).

### CODE REVIEW

**Comment by rusty-snake** [comment]
Now we have `noexec ${HOME}` and `noexec /tmp` duplicated in most profiles, IMHO it would make sense to remove `noexec ${HOME}` and `noexec /tmp` from these profiles.

**Comment by smitsohu** [comment]
@rusty-snake Thanks, you're right. I clean it up later today.

**Comment by smitsohu** [comment]
Done :smile:

File: etc/electron.profile [hunk 1]
```diff
@@ -7,6 +7,7 @@ include electron.local
 include globals.local
 
 include disable-common.inc
+include disable-exec.inc
```
Comment (Fred-Barclay) [inline etc/electron.profile]: This didn't already have noexec though, right?

Comment (smitsohu) [reply etc/electron.profile]: Right! Is it possible electron.profile is a bit too restrictive anyway?

Comment (Vincent43) [reply etc/electron.profile]: It could be. I think we should remove `disable-exec.inc` from it and do other changes in separate PR.

Comment (smitsohu) [reply etc/electron.profile]: I'll remove it in a second commit.

**Comment by Vincent43** [comment]
The next step would be to replace `noexec $HOME` and `noexec /tmp` with `disable-exec.inc ` in rest of profiles.

**Pull request merged**